library/library_info.mli

```diff
@@ -0,0 +1,15 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+open Names
+
+type t = Comment of string | Deprecation of Deprecation.t
+
+val warn_library_info : ?loc:Loc.t -> DirPath.t * t -> unit
```

Comment:
Does this really deserves its own file? If yes, why not adding a top comment with a sentence explaining what the file is about?

Reply:
> Does this really deserves its own file

Not necessarily, but it somehow provides an API for library file informations.